manifests/create-cert.sh

```diff
@@ -14,11 +14,26 @@ detailed explantion and additional instructions.
 
 The server key/cert k8s CA cert are stored in a k8s secret.
 
+       --namespace        Namespace where webhook service and secret reside.
 EOF
     exit 1
 }
 
-namespace=default
+while [[ $# -gt 0 ]]; do
+    case ${1} in
+        --namespace)
+            namespace="$2"
+            shift
+            ;;
+        *)
+            usage
+            ;;
+    esac
+    shift
+done
+
+
+[ -z ${namespace} ] && namespace=default
```

Comment:
Use `namespace=${namespace:-default}` instead.

Reply:
I think we should default to `tidb-admin` namespace.

Reply:
done